src/main.rs

```diff
@@ -32,33 +40,47 @@ static ES_ES: &str = include_str!("./locales/es-ES.json");
 fn App(cx: Scope) -> Element {
     use_init_i18n(
         cx,
-        "es-ES".parse().unwrap(),
-        "es-ES".parse().unwrap(),
+        "es-ES".parse().expect("can't parse es-ES language"),
+        "es-ES".parse().expect("can't parse es-ES language"),
         || {
-            let en_us = Language::from_str(EN_US).unwrap();
-            let es_es = Language::from_str(ES_ES).unwrap();
+            let en_us = Language::from_str(EN_US).expect("can't get EN_US language");
+            let es_es = Language::from_str(ES_ES).expect("can't get ES_ES language");
             vec![en_us, es_es]
         },
     );
 
+    if let Some(window) = window() {
```

Comment:
Thinking you can reduce nesting using a block that returns an option and inside you can use `?`(it's not only for Results).
e.g.
```rust
let Some(parent) = {
    window()?.document()?.get_element_by_id("...")?.parent_node()
}
```
It might need some type annotations but I think it should work